docs/platform_admin_guide.md

```diff
@@ -49,31 +49,27 @@ All these values are compared with one regex pattern:
 `^([+]?[0-9.]+)([eEinumkKMGTP]*[+]?[0-9]*)$`
 
 Learn more about namespace quotas in 
-[K8S documentation](https://kubernetes.io/docs/concepts/policy/resource-quotas/#compute-resource-quota) 
-and managing compute resources (https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/).  	
- 
+[K8S documentation](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/).  	
  
 Additionally, it is possible to specify how many endpoints can be created within a tenant. 
 To do that, use a parameter `maxEndpoints`.
-
-All the quota parameters are optional.
  
 Check for the API example on [Management API doc](../management) and review [example CLI](../scripts).
 
 
-## Listing tenants
+### List tenants
 
-Platform admin can list names of tenants by using GET API request on `https://<management-api-address>/tenants` address.
+Platform Admin can list names of tenants using GET API request on `https:/<management-api-address>/tenants` address.
```

Comment:
Why did you delete one slash in "https:/<management-api-address>/tenants"? Was it on purpose?